### PR TITLE
KingSuper pledge

### PR DESCRIPTION
**Pull request by aditya-manit**

### CODE REVIEW

File: KingSuper/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Agoric Validator Pledge
+
```
Comment (jeetraut) [inline KingSuper/README.md:2]:
```suggestion
[KingSuper](https://main.explorer.agoric.net/validator/agoricvaloper1fy8r6zq3dde6n7ak6eqy5zsrp3s7sz833dmv32)
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Review by jeetraut — COMMENTED** [review]
Looks good!

**Comment by dckc** [comment]
@aditya-manit is doing a great job helping to review 1 BLD requests (#80).

**Comment by aditya-manit** [comment]
Thanks @dckc :D :D

**Pull request merged**